CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## Unreleased changes
+## 9.18
```

Comment:
Can you add `Pass through label params from textarea component (PR #508)` to this? Sorry I forgot.